client/src/i18n/en-US.json

```diff
@@ -365,7 +365,7 @@
       },
       "coordinator": {
         "title": "Review Coordination",
-        "byline": "{type_name} you are coordinating for reviewers",
+        "byline": "{type_name} you are review coordinating",
```

Comment:
This can result in "Reviews you are review coordinating" which is repetitive. It also loses emphasis on the people the coordinator is working with.

Reply:
1. Would {type_name} display the submission title, or just "article"?
2. "Coordinating for reviewers" is not an accurate description of what review coordinators do. "Review coordinating" is the phrase we use.

Reply:
1. `{type_name}` refers to the type of the table that this byline will appear within. In this case, I believe `{type_name}` always maps to "Reviews". I think you've pointed out a good redundancy here because, if this always maps to "Reviews," then I don't think the `{type_name}` variable is necessary. However, I'll need to double-check. I'll see if I can refactor this so that `{type_name}` is just replaced by the static text "Reviews" so you don't have to worry about the overcomplicated indirection here.

Reply:
I was finally able get around to confirming that the `{type_name}` variable is redundant, and so I've removed it. This text now reads, "Reviews you are review coordinating." 

Would you like to further modify this?

Reply:
Shelby responded separately that there is no further need to modify this.